### ISSUE TITLE
VTR does not display tabs in private window

### ISSUE THREAD

**Issue by adelton** [Croydon/vertical-tabs-reloaded#167]
Add-on version: 0.9.4
Firefox version: firefox-57.0.1-1.fc27.x86_64
OS + OS-Version: Fedora 27
Other tab + user interface related add-ons: none but I use `#tabbrowser-tabs { visibility: collapse !important; }` per https://www.ghacks.net/2017/09/27/tree-style-tab-is-a-webextension-now/

When New Window is created with Ctrl+N, the sidebar is present and there is one "New Tab" tab and things work as expected.

When New Private Window is created via Ctrl+Shift+P, the sidebar is not present and when I bring it up with Ctrl+Shift+V, it is empty and creating new tabs with Ctrl+T does not show the tabs.

With Tree Style Tab, new private window does not have the sidebar by default either and it has to be enabled with F1 but it then shows the tabs just fine.

**Comment by nathantypanski** [comment]
![2018-02-10-142821_2880x1800_scrot](https://user-images.githubusercontent.com/1385140/36065904-f4204bde-0e6e-11e8-8875-4aafb046aa56.png)

I get this also on Linux. Works on OS X.

```
$ firefox --version
Mozilla Firefox 58.0.2
$ uname -a
Linux whitebook 4.15.2-2-ARCH #1 SMP PREEMPT Thu Feb 8 18:54:52 UTC 2018 x86_64 GNU/Linux
````

**Comment by Croydon** [comment]
This is blocked by https://bugzilla.mozilla.org/show_bug.cgi?id=1380809

Because browser.extension.getBackgroundPage() doesn't work in private windows.   https://bugzilla.mozilla.org/show_bug.cgi?id=1329304

**Comment by Croydon** [comment]
Mozilla isn't going to fix these issues in Firefox 60 anymore (best hope is now for 61).

Since Firefox 60 is going to become the next ESR I will try to find an tolerable workaround for this. I want to actively support Firefox 60 ESR for at least some time and I also want to make VTR work in private mode as soon as possible again.

Results are duplicate of logic in code and potential drop in performance...

I really wish they would support Firefox 52 ESR for longer and wait with a new ESR release till the APIs are more mature.

**Comment by Croydon** [comment]
Private mode will work in general in 0.11.0.

Editing the options of VTR will continue to be only possible in non-private windows.
